scripts/dev/build_release.py

```diff
@@ -357,6 +417,8 @@ def build_mac(
     call_tox(f'pyinstaller-64{"-qt6" if qt6 else ""}', '-r', debug=debug)
     utils.print_title("Patching .app")
     patch_mac_app(qt6=qt6)
```

Comment:
the `qt6` argument here implies that this script build applications targeted at both PyQt5 and PyQt6. The new `fixup_qtwebengine()` method seems to only target PyQt6. Should it handle both? Should it only run for PyQt6?